### PR TITLE
Disable unwanted repos for upgrade

### PR DESCRIPTION
**Pull request by jyejare**
PR includes:
1. Disables all repos first
2. Enables base OS repo
3. And enables/sets latest satellite repo

 .... for satellite upgrade !

### CODE REVIEW

**Comment by jyejare** [comment]
Here is the fool proof that It worked:
https://satellite-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Satellite%206/job/satellite6-upgrader/71/consoleFull
Ran jenkins job from my branch 'repo_fix' on rhel 6 machine.

**Comment by jyejare** [comment]
Thanks @kbidarkar for helping in finding the issue.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -2171,19 +2171,23 @@ def satellite6_upgrade(admin_password=None):
...
-    # Add Sat6 repo from latest compose
-    satellite_repo = StringIO()
-    satellite_repo.write('[sat6]\n')
-    satellite_repo.write('name=satellite 6\n')
-    satellite_repo.write('baseurl={0}\n'.format(base_url))
-    satellite_repo.write('enabled=1\n')
-    satellite_repo.write('gpgcheck=0\n')
-    put(local_path=satellite_repo, remote_path='/etc/yum.repos.d/sat6.repo')
-    satellite_repo.close()
+    else:
```
Comment (kbidarkar) [inline automation_tools/__init__.py]: Good Catch. :+1:

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -2171,19 +2171,23 @@ def satellite6_upgrade(admin_password=None):
...
+        # Add Sat6 repo from latest compose
+        satellite_repo = StringIO()
```
Comment (kbidarkar) [inline automation_tools/__init__.py]: I think we need to use spaces instead of tabs for automation-tools too. Please use spaces.

**Comment by kbidarkar** [comment]
ACK, pending minor nitpicks.

File: automation_tools/__init__.py (hunk 2 above)
Comment (jyejare) [inline automation_tools/__init__.py]: I tried adding spaces instead of tabs from eclipse as well as from VI editor. No luck :(

**Comment by elyezer** [comment]
ACK

**Pull request merged**